src/pages/hyunjunlee/Login/login-hyunjun.js

```diff
@@ -1,9 +1,73 @@
+/* eslint-disable jsx-a11y/anchor-is-valid */
 import React from 'react';
+import { withRouter } from 'react-router-dom';
+import '../../hyunjunlee/Login/lgh.scss';
 
 class LoginHyun extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      IdValue: '',
+      PwValue: '',
+    };
+  }
+  goToMain = () => {
+    if (this.state.IdValue.includes('@') && this.state.PwValue.length > 4)
+      this.props.history.push('./main-hyunjun');
+    fetch('http://10.58.0.138:8000/users/signin', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.IdValue,
+        password: this.state.PwValue,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log('결과: ', result));
+  };
```

Comment:
```suggestion
  goToMain = () => {
    const { id, pw } = this.state
    
    if (id.includes('@') && pw.length > 4) {
      fetch('http://10.58.0.138:8000/users/signin', {
        method: 'POST',
        body: JSON.stringify({ email: id, password: pw }),
      })
        .then(response => response.json())
        .then(result => {
          this.props.history.push('./main-hyunjun');
        });
    }
  };
```
- if 문을 한줄로 쓸 수 있기는 하지만 그 한줄이 길어져 두줄이 됐을을 경우에는 꼭 중괄호를 써주세요
- 로그인 관련 모든 로직들이 성공했을 때 main으로 넘어가야 하지 않을까요?
- 구조분해 할당 해주세요!